pages/api-reference/funding-sources/create-funding-source-for-customer.mdx

```diff
@@ -13,13 +13,24 @@ There are two methods available for adding a bank or credit union account to a C
 
 Before a Dwolla account or Dwolla API Customer is eligible to transfer money from their bank or credit union account they need to verify ownership of the account, either via Instant Account Verification (IAV) or micro-deposits. For more information on bank account verification, reference this [funding source verification](https://developers.dwolla.com/resources/funding-source-verification.html) resource article.
 
+[Verified Customers](https://developers.dwolla.com/concepts/customer-types#customer-types) also have the ability to add Virtual Account Numbers, or `VANs`. A `VAN` can only be used outside of the Dwolla Platform to route ACH payments both to and from the Dwolla Balance.
+
+<AlertBar variation="warning">
+  Because VANs behave as a route to a Dwolla Balance, only{" "}
+  <a href="/concepts/customer-types#verified-customer">Verifed Customers</a> and
+  Dwolla Master Accounts are eligible to create one. VANs count toward a total
+  funding source limit and is a premium feature. To learn more about pricing and
+  enabling this functionality, please{" "}
+  <a href="hhttps://www.dwolla.com/contact?b=apidocs">contact Sales</a>.
+</AlertBar>
+
 Customers can have a maximum of 2 active funding sources.
 
-### HTTP request
+### HTTP request - bank account funding source
```

Comment:
Let's use "bank funding source" to stay consistent with other mentions of this term.